create-owner.js.j2

```diff
@@ -0,0 +1,100 @@
+/**
+ * This Event Handler is expecting a message with atleast those attributes:
+ * {
+ *     "username": "bob",
+ *     "password": "password"
+ * }
+ */
+
+{{ jsrequire }}
+
+{{ jsauth }}
+
+/**
+ * Check if an owner exists on the mnubo's platform
+ *
+ * https://smartobjects.mnubo.com/documentation/api_ingestion.html#get-api-v3-owners-exists-username
+ * @param {string} ownersUri URI to the mnubo's Object API
+ * @param {object} token token retrieved from the mnubo's API
+ * @param {string} username the owner's username
+ */
+function checkOwnerExistence(ownersUri, token, username) {
+    // eslint-disable-next-line
+    console.log('Checking if owner exists on mnubo');
+
+    const existsUri = `${ownersUri}/exists/${username}`;
+    const httpOptions = {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${token.access_token}`
+        },
+    };
+    return XHR.fetch(existsUri, httpOptions).then((resp) => {
```

Comment:
I would insert an empty line before the return like you did in `onRequest` function.